internal/driver/mobile/app/app.go

```diff
@@ -1,16 +1,13 @@
-// Copyright 2014 The Go Authors. All rights reserved.
```

Comment:
I don't think it's OK to just strip copyright... perhaps add us to it rather than removing all?

Reply:
Removing the header is to make it clear that it has largely deviated from the original distribution. Adding a README in this folder is optimal to keep the attribution so that we don't violate the BSD license. Such as: https://github.com/fyne-io/fyne/blob/master/cmd/fyne/internal/mobile/README.txt

Reply:
But copyright is not the same as licensing - when we fork the code we did not change it's origination or revoke the copyright of it's original holders...